launchlib/launcher.go

```diff
@@ -91,6 +91,22 @@ func CompileCmdFromConfig(staticConfig *StaticLauncherConfig, customConfig *Cust
 	return createCmd(executable, args, env)
 }
 
+func MkDirs(dirs []string) error {
+	isDirMatcher := regexp.MustCompile(`^[A-Za-z0-9]+(/[A-Za-z0-9]+)*$`).MatchString
+	for _, dir := range dirs {
+		if !isDirMatcher(dir) {
+			return fmt.Errorf("Cannot create directory with non [A-Za-z0-9] characters: %s", dir)
+		}
+
+		fmt.Printf("Creating directory: %s\n", dir)
+		err := os.MkdirAll(dir, 0700)
```

Comment:
Minor matter of preference, but if a function returns only an error that you check just once, you can inline the call and check (and this is often done in Go):

```go
if err := os.MkdirAll(dir, 0700); err != nil {
  return err
}
```